### PR TITLE
CODE RUB: Update Image Link To Be Markdown

### LINKED ISSUE #138: CODE RUB: Update Image Link To Be Markdown

### PR DESCRIPTION
**Pull request by ElbekDeveloper**
Closes #138

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,6 @@
-# OpenAI.NET
+# Standard.AI.OpenAI
 
-<div align=center>
-	<img width="300" src="https://raw.githubusercontent.com/hassanhabib/OpenAI.NET/main/OpenAI.NET/artificial-intelligence.png" />
-</div>
-
-<br />
+![Standard.AI.OpenAI Icon](https://raw.githubusercontent.com/hassanhabib/OpenAI.NET/main/OpenAI.NET/artificial-intelligence.png)
```
Comment (hassanhabib) [inline README.md:3]: @ElbekDeveloper is this gonna be centered?

Comment (ElbekDeveloper) [reply README.md:3]: No, markdown doesn't allow styling.

Comment (ElbekDeveloper) [reply README.md:3]: That's why I added 300px icon so that I can update the image to be smaller in the next PR. But, it will stay aligned left

Comment (hassanhabib) [reply README.md:3]: @ElbekDeveloper we need to make it centered. Please find a way.

Comment (cjdutoit) [reply README.md:3]: @ElbekDeveloper / @hassanhabib 
`<p align="center">` is not allowed in markdown.

The only way to center the icon is to create a transparent banner with a resolution of 1280x640px where the icon is centered.

This is also the same dimentions sugested by GitHub when you go to Settings > Social Preview

Comment (mabroukmahdhi) [reply README.md:3]: > @ElbekDeveloper / @hassanhabib `<p align="center">` is not allowed in markdown.
> 
> The only way to center the icon is to create a transparent banner with a resolution of 1280x640px where the icon is centered.
> 
> This is also the same dimentions sugested by GitHub when you go to Settings > Social Preview

Agree with @cjdutoit . I was looking for a way to get it centred, but no way with markdown code.

Comment (hassanhabib) [reply README.md:3]: @cjdutoit @mabroukmahdhi @ElbekDeveloper I saw some other repos put in a really wide header logo - is this the way to go?

here's an example: https://github.com/NuGetPackageExplorer/NuGetPackageExplorer

Comment (glhays) [reply README.md:3]: @hassanhabib
I am working a possible "meaning if acceptable" solution as per your comment example above. I Should have recognized the bigger picture here. You know that light that goes on after you have spent a exorbitant amount of muscle to move something and bang the light hits you why didn't you just do as suggested. The breadcrumb was there all along.

**Pull request closed without merging**